src/TableRow.tsx

```diff
@@ -76,7 +76,7 @@ class TableRow<T = any> extends React.PureComponent<ITableRowProps<T>> {
         rowIndex,
         expandIcon: column.key === expandColumnKey && expandIcon,
       };
-      return <CellRenderer {...cellProps} />;
+      return <CellRenderer {...cellProps} key={columnIndex}/>;
```

Comment:
Do you get any warning in in the website, becase the cellProps might have a key already, and then I dont know which one takes precedence

Reply:
Hmm. Didn't see warning yet, but some examples are not working, for example, `RowSpan`, `ColumnSpan`. Not sure if it's related to this one.